### ISSUE TITLE
Tiqr wants to write to the "logs" directory

### ISSUE THREAD

**Issue by pmeulen** [OpenConext/Stepup-tiqr#236]
A HTTP GET to `/health` endpoint logs an:

`[critical] Uncaught Exception: Unable to write in the "logs" directory (/var/www/html/var/log).`

Version: Tiqr 4.1.0-beta1 on TEST2 environment

The logging configuration looks OK: https://github.com/OpenConext/Stepup-tiqr/blob/4.1.0-beta1/config/packages/monolog.yaml#L11

I did test deploying 4.0.7-beta1 to the TEST2 environment, this version works.
4.1.0-beta1 does not have this issue when started the devconf.

It is unclear what causes this error, please investigate.

Expected: Components never write to an application logs directory, use the monolog configuration.

**Comment by johanib** [comment]
I did some debugging together with Pieter and subsequent testing in my devbox to validate what happened.

I think this behaviour is not changed in this release but it triggered because a new parameter was missing in `parameters.yaml`

I assume the cli user in the container is the root user, which does have access to the `var/log` directory.

What I think usually happens is:
* Container gets built/deployed
* From within the container something like `./bin/console cache:clear --env=prod` get executed by the root user, which can write to the `var/log` dir.
* This builds and caches the Symfony container
* Then, when the `/health` endpoint gets called, all is well

What happend this time:
* Container gets built/deployed
* From within the container, `./bin/console cache:clear --env=prod` gets executed, but it fails with a missing parameter error, the container does not get built and cached
* When `/health` gets called, it tries to build the cache/container, and shows the "Unable to write" error.

Symfony never writes anything to the `var/log` dir.

The `Exception: Unable to write in the` exception from the web is logged to the docker logs.
From the CLI, it gets outputted to the cli.

I do think its weird Symfony checks the permissions without needing the `var/log` dir for actual writing, but it does seem to be built in as a hard requirement. Looking at the http-kernel code, this cannot be circumvented.
A workaround can be to change it to `/tmp` for example, but for our usecase this does not make a difference.

When the `var/log` dir is writable and an error occurs during bootstrapping, for example, a parameter is missing, the error gets written to the docker logs and not to the `var/log` dir.

So in summary, Symfony does have a hard requirment on a writable log dir, even if it will probably never use it. 
It will cause issues and this error if the logs dir is not writable.

---

https://github.com/symfony/symfony/blob/7.3/src/Symfony/Component/HttpKernel/Kernel.php#L603

**Comment by pmeulen** [comment]
We're going to solve this in the basecontainer and / or Ansible-deploy
